package.json

```diff
@@ -126,10 +126,14 @@
     "compile": "tsup ./src/extension.ts --external vscode --minify",
     "watch": "tsup ./src/extension.ts --external vscode --watch --sourcemap",
     "test": "vitest run --config vite.global-config.ts",
+    "test:e2e": "rimraf .test-extensions && rimraf out && tsc -p ./ && extest setup-and-run './out/tests/e2e/*.test.js' --extensions_dir .test-extensions",
```

Comment:
We are building with tsup, not tsc

Reply:
I am having some problems configuring mocha to process typescript files directly. Can we leave it as-is for now, and then I will explore a better solution in the next PR?